### DISCUSSION TITLE
ScriptReference/RuntimeInitializeLoadType

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#497] [Notes]
# ScriptReference/RuntimeInitializeLoadType

https://docs.unity3d.com/ScriptReference/RuntimeInitializeLoadType.html

**Comment by docscomments** [comment]
**Comment by `@DietChugg`:**  

`RuntimeInitializeLoadType` does not get called on static generic classes, obviously there will be no appropriate generic argument to use.

**Reply by ZZumMallen** [reply]
#pragma warning disable UDR0001
